### ISSUE TITLE
People shown multiple times in completion when mentioning them in chat

### ISSUE THREAD

**Issue by dusatvoj** [RocketChat/Rocket.Chat#19438]
### Description:

When typing a message and trying to mention somebody is everybody shown multiple times in autocompletion. 

### Steps to reproduce:

1. Write a message
2. Start typing `@` and some characters of somebody's username
3. Sometimes are users shown multiple times

### Expected behavior:

Everybody are shown only once in completion

### Actual behavior:
- Sometimes is it like this: ![image](https://user-images.githubusercontent.com/11560794/97904006-2fae5a00-1d40-11eb-9e30-eb94c2f5b940.png)
- But sometimes is it like this: ![image](https://user-images.githubusercontent.com/11560794/97904473-d2ff6f00-1d40-11eb-8f9b-eaf171614139.png)

Can't understand when is which behavior

### Server Setup Information:

- Version of Rocket.Chat Server: 
- Operating System: Linux Debian 9/10
- Deployment Method: tar
- Number of Running Instances: 3
- DB Replicaset Oplog: Yes (`USE_NATIVE_OPLOG=true` bcs of DB permissions and not completed docs)
- NodeJS Version: `v12.18.4`
- MongoDB Version: `6.14.8`

### Client Setup Information

- Desktop App or Browser Version: `rocketchat-desktop 3.0.3-1`
- Operating System: Arch Linux, AUR package but seen in Windows desktop app too.

**Comment by scrivna** [comment]
Came here to report this, having the same issue with 3.7.1

**Comment by ankar84** [comment]
Duplicate of https://github.com/RocketChat/Rocket.Chat/issues/19225

**Comment by MartinSchoeler** [comment]
As @ankar84 mentioned this is a duplicate of #19225 Please continue the discussion there. Thanks!
